hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/TransactionMethod.java

```diff
@@ -29,6 +29,7 @@
  * <p>FUTURE WORK: ThreadSafe annotation missing in spotbugs annotations but should be added to
  * class
  */
+/*@ThreadSafe*/
```

Comment:
Should we add a @ThreadSafe annotation in the repo until we have a basic one for all sources / repos?

Reply:
I would be good with this suggestion, but I didn't really want to do it in this PR since it is already so big